packages/hardware-ledger/src/transformers/certificates.ts

```diff
@@ -245,18 +289,116 @@ const poolRetirementCertificate: Transform<
   };
 };
 
-const toCert = (cert: Cardano.Certificate, context: LedgerTxTransformerContext) => {
+const checkDrepPublicKeyAgainstCredential = (
```

Comment:
@mchappell this one would always throw since Crypto.Ed25519PublicKeyHex expects to get 64 char long string and we are passing hash that is Crypto.Hash28ByteBase16 that is 56 char?

Reply:
resolved